.github/pull_request_template.md

```diff
@@ -0,0 +1,55 @@
+## Title (delete after following this rule)
+
+Your PR title should be prefixed with the Jira ticket ID, e.g. "ADX-123 very important feature".
+
+## Description
+
+Please include a summary of the change(s).
+You should also include motivation: why was this change required and what problem it solves.
+
+Don't forget to reference all other pull requests which are associated to this one below, especially if merging need to be coordinated.
+E.g.
+
+relates fjelltopp/repository_name#issue_number
+
+## Dependency Changes (delete if not applicable)
+
+List any dependencies that have been removed/added/changed in this pull request.
+Has the pipfile.lock or yarn.lock been updated?
+
+## Testing (delete if not applicable)
+
+Please describe the new tests that have been added for this change along with any additional instructions required to reproduce a successful test and/or a list of any relevant details for your test configuration.
+
+## Documentation (delete if not applicable)
+
+Please describe any new technical documentation that has been added to Confluence for this change.
+
+## Environment Changes (delete if not applicable)
+
+Have any environment variable changes been committed as part of this pull request?
+Is there a matching PR in fjelltopp/fjelltop-infrastructure for these changes?
+Have any README or Confluence install instructions been updated to reflect this change?
+
+## Internationalisation and Localisation (delete if not applicable)
+
+Do any UI changes require internationalisation (i18n) or localisation (l10n) changes.
+If new translations are required, have you created new Jira tickets for each language and informed the appropriate project manager?
+
+## Checklist
+
+Put an `x` in the boxes that apply to this pull request (you can also fill these out after opening the pull request).
+You may not need to check all boxes.
+
+- [ ] The Jira ticket for this issue has been updated to "Ready to Review" or equivalent.
+- [ ] I have developed these changes in discussion with the appropriate project manager.
+- [ ] Any UI changes have been approved by an appropriate project manager.
+- [ ] My code follows the general Fjelltopp documentation (see Confluence).
+- [ ] I have made corresponding changes to the Fjelltopp documentation (see Confluence).
+- [ ] I have rebased this branch with master.
+- [ ] New dependency changes have been committed.
+- [ ] New and existing unit tests pass locally with my changes.
```

Comment:
"unit tests" > "automated tests"
Tests also need to be executed automatically with github actions.